src/main/java/org/ohdsi/cohortcharacterization/CCQueryBuilder.java

```diff
@@ -134,27 +136,36 @@ private String renderCustomAnalysisDesign(FeatureAnalysis fa, Integer cohortId)
 		);
 	}
 
-	private List<String> getQueriesForCustomDistributionAnalyses(final Integer cohortId) {
+	private List<String> getQueriesForCustomDistributionAnalyses(final Integer cohortId, CohortCharacterizationStrata strata) {
+
+		Long strataId = Objects.nonNull(strata) ? strata.getId() : 0L;
```

Comment:
the strata is going to be a string eventually, so maybe it is better to set String type here 
```
String  strataId = Objects.nonNull(strata) ? String.valueOf(strata.getId()) : "0";
```